config/default.yml

```diff
@@ -119,6 +119,15 @@ Rails/BulkChangeTable:
   Include:
     - db/migrate/*.rb
 
+Rails/ContentTag:
+  Description: 'Use `tag` instead of `content_tag`.'
+  Reference:
+    - 'https://github.com/rails/rails/issues/25195'
+    - 'https://api.rubyonrails.org/classes/ActionView/Helpers/TagHelper.html#method-i-content_tag'
+  Enabled: true
+  Safe: true
```

Comment:
Ah, can you omit `Safe: true` because it is safe by default?

Reply:
Oh, I see.